v2/managed-io-to-managed-io/src/main/java/com/google/cloud/teleport/v2/templates/ManagedIOToManagedIO.java

```diff
@@ -0,0 +1,264 @@
+            + "The template uses Apache Beam's Managed API to provide a unified interface for "
+            + "configuring different I/O connectors through simple configuration maps.",
+    optionsClass = Options.class,
+    flexContainerName = "managed-io-to-managed-io",
+    documentation = "https://cloud.google.com/dataflow/docs/guides/managed-io",
+    contactInformation = "https://cloud.google.com/support",
+    requirements = {
+      "The source and sink configurations must be valid for the specified connector types.",
+      "Required permissions for accessing the source and sink systems."
+    },
+    streaming = false)
+public class ManagedIOToManagedIO {
+
+  /**
+   * The {@link Options} class provides the custom execution options passed by the executor at the
+   * command-line.
+   */
+  public interface Options extends PipelineOptions {
+
+    @TemplateParameter.Text(
+        order = 1,
+        description = "Source Managed I/O connector type",
+        helpText =
+            "The type of Managed I/O connector to use as source. "
+                + "Supported values: ICEBERG, ICEBERG_CDC, KAFKA, BIGQUERY.")
+    String getSourceConnectorType();
+
+    void setSourceConnectorType(String value);
+
+    @TemplateParameter.Text(
+        order = 2,
+        description = "Source configuration (JSON)",
+        helpText =
+            "JSON configuration for the source Managed I/O connector. "
+                + "The configuration format depends on the connector type. "
+                + "For example, for KAFKA: {\"bootstrap_servers\": \"localhost:9092\", \"topic\": \"input-topic\", \"format\": \"JSON\"}")
+    String getSourceConfig();
+
+    void setSourceConfig(String value);
+
+    @TemplateParameter.Text(
+        order = 3,
+        description = "Sink Managed I/O connector type",
+        helpText =
+            "The type of Managed I/O connector to use as sink. "
+                + "Supported values: ICEBERG, KAFKA, BIGQUERY. "
+                + "Note: ICEBERG_CDC is only available for reading.")
+    String getSinkConnectorType();
+
+    void setSinkConnectorType(String value);
+
+    @TemplateParameter.Text(
+        order = 4,
+        description = "Sink configuration (JSON)",
+        helpText =
+            "JSON configuration for the sink Managed I/O connector. "
+                + "The configuration format depends on the connector type. "
+                + "For example, for BIGQUERY: {\"table\": \"project:dataset.table\"}")
+    String getSinkConfig();
+
+    void setSinkConfig(String value);
+
+    @TemplateParameter.Boolean(
+        order = 5,
+        optional = true,
+        description = "Enable streaming mode",
+        helpText =
+            "Whether to run the pipeline in streaming mode. "
+                + "This is only supported for certain connector combinations. "
+                + "Default is false (batch mode).")
+    boolean isStreaming();
```

Comment:
Can we infer this from the source?

Reply:
how urgent do you want this? We could improve this later by checking the source but our regular templates usually have this option.

Reply:
I think most default to the correct option - https://cloud.google.com/dataflow/docs/guides/templates/provided-templates#streaming-templates - even if they expose something else. I also don't see this exposed for other templates.

IMO it is easy enough to do that we should just include it